claims.go

```diff
@@ -29,14 +36,20 @@ type StandardClaims struct {
 // There is no accounting for clock skew.
 // As well, if any of the above claims are not in the token, it will still
 // be considered a valid claim.
-func (c StandardClaims) Valid() error {
+func (c StandardClaims) Valid(opts *ValidationOptions) error {
 	vErr := new(ValidationError)
 	now := TimeFunc().Unix()
 
+	// Get leeway out of opts (if present)
+	var leeway int64
+	if opts != nil {
+		leeway = opts.Leeway
```

Comment:
If leeway is set to '-1' then we should perhaps set to '0'

Reply:
Perhaps even better to change the type of `uint64` to avoid mistakes? and simply do`int64(opts.Leeway)` here